tests/storage/test_azure_storage.py

```diff
@@ -36,16 +36,39 @@ def test_serialize_azure_storage():
     assert serialized_storage["type"] == "Azure"
 
 
-def test_blob_service_client_property(monkeypatch):
+@pytest.mark.parametrize(
+    "secret_name,secret_arg",
+    [
```

Comment:
I have a comment below, but similar here I think the parameters should have different values so that the assertions are for sure evaluated correctly.